### ISSUE TITLE
[1.16.5] Error when make a world : WorldGenSettings: Value -1 outside of range

### ISSUE THREAD

**Issue by quentin452** [quentin452/1.7.10-Forge-renewed-v1#210]
```
[30Jan2023 01:33:40.866] [Server thread/ERROR] [net.minecraft.world.storage.ServerWorldInfo/]: WorldGenSettings: Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]; Value -1 outside of range [0:4096]

**Comment by quentin452** [comment]
seem to be fixed in V0.81 Hotfix by removing idk

**Comment by quentin452** [comment]
fixed in V0.81 Hotfix by removing dalek mods + addons

**Comment by quentin452** [comment]
caused by ender cataclysm , but is a good mod so not deleted
